src/main/java/fi/metatavu/metaform/server/rest/translate/AulditLogEntryTranslator.java

```diff
@@ -0,0 +1,42 @@
+package fi.metatavu.metaform.server.rest.translate;
```

Comment:
Is this class really used somewhere? If it does, it should be finalized and the typo in file name should be fixed